content/docs/tutorials/get-started/index.md

```diff
@@ -1,51 +1,59 @@
-# Get Started with DVC!
+# Get Started
 
-DVC is a **data version control**, pipeline management, and experiment
-management tool that brings existing engineering toolsets and practices to data
-science and machine learning. In this guide we will introduce the basic features
-and concepts of DVC step by step.
```

Comment:
So no short intro after all? There's really no other better place to introduce what DVC is. Unless we review/update the home page (https://dvc.org/) — which needs some updating per 1.0 anyway.

Reply:
> which needs some updating per 1.0 anyway

1.0 does not directly affect it I would say

> So no short intro after all? 

I'm simplifying everything for now. Also, I just don't think this single paragraph is very useful to be honest. I won't help people understand it better. We'll figure out eventually how to deal with it.

Reply:
OK. I guess /doc is a better place anyway.

> 1.0 does not directly affect it I would say (home page)

`dvc run` needs -n for staters 😬 

And scrolling down, maybe the features could match the bullets in this new Get Stared.

Reply:
> dvc run needs -n for staters

it's fine to keep at is in the animation, also we'll be redoing it completely (not related to 1.0)

> And scrolling down, maybe the features could match the bullets in this new Get Stared.

potentially yes, it can benefit from mentioning params, plots. But the way landing page is done now it's a minor change. High level structure is not affected.